README.md

```diff
@@ -0,0 +1,219 @@
+- Stack Group support by bundling related Stacks into logical groups (e.g. dev and prod)
+- Stack Group-level commands, such as creating multiple Stacks with a single command
+- Fast, highly parallelised builds
+- Built in support for working with Stacks in multiple AWS accounts and regions
+- Infrastructure visibility with meta-operations such as Stack querying protection
+- Support for inserting dynamic values in templates via customisable Resolvers
+- Support for running arbitrary code as Hooks before/after Stack builds
+
+# Benefits
+
+- Utilises cloud-native Infrastructure as Code engines (CloudFormation)
+- You do not need to manage state
+- Simple templates using popular templating syntax - Yaml & Jinja
+- Powerful flexibility using a mature programming language - Python
+- Easy to integrate as part of a CI/CD pipeline by using Hooks
+- Simple CLI and API
+- Unopinionated - Sceptre does not force a specific project structure
+
+# Install
+
+`$ pip install sceptre`
+
+More information on installing sceptre can be found in our [Installation Guide](https://sceptre.cloudreach.com/latest/docs/install.html)
+
+# Example
+
+Sceptre organises Stacks into "Stack Groups". Each Stack is represented by a
+YAML configuration file stored in a directory which represents the Stack Group.
+Here, we have two Stacks, `vpc` and `subnets`, in an Stack Group named `dev`:
+
+```bash
+$ tree
+.
+├── config
+│   └── dev
+│        ├── config.yaml
+│        ├── subnets.yaml
+│        └── vpc.yaml
+└── templates
+    ├── subnets.py
+    └── vpc.py
+```
+
+We can create a Stack with the `create` command. This `vpc` Stack contains
+a VPC.
+
+```bash
+$ sceptre create dev/vpc.yaml
+
+dev/vpc - Creating stack dev/vpc
+VirtualPrivateCloud AWS::EC2::VPC CREATE_IN_PROGRESS
+dev/vpc VirtualPrivateCloud AWS::EC2::VPC CREATE_COMPLETE
+dev/vpc sceptre-demo-dev-vpc AWS::CloudFormation::Stack CREATE_COMPLETE
+```
+
+The `subnets` Stack contains a subnet which must be created in the VPC. To do
+this, we need to pass the VPC ID, which is exposed as a Stack output of the
+`vpc` Stack, to a parameter of the `subnets` Stack. Sceptre automatically
+resolves this dependency for us.
+
+```bash
+$ sceptre create dev/subnets.yaml
+dev/subnets - Creating stack
+dev/subnets Subnet AWS::EC2::Subnet CREATE_IN_PROGRESS
+dev/subnets Subnet AWS::EC2::Subnet CREATE_COMPLETE
+dev/subnets sceptre-demo-dev-subnets AWS::CloudFormation::Stack CREATE_COMPLETE
+```
+
+Sceptre implements meta-operations, which allow us to find out information
+about our Stacks::
+
+```bash
+$ sceptre describe resources dev dev/subnets
+
+- LogicalResourceId: Subnet
+  PhysicalResourceId: subnet-445e6e32
+  dev/vpc:
+- LogicalResourceId: VirtualPrivateCloud
+  PhysicalResourceId: vpc-c4715da0
+```
+
+Sceptre provides Stack Group level commands. This one deletes the whole `dev`
+Stack Group. The subnet exists within the vpc, so it must be deleted first.
+Sceptre handles this automatically::
```

Comment:
I think we should add additional information over the following behavior:

```bash
$ tree
.
├── config
│   └── dev
│        ├── config.yaml
│        ├── subnets.yaml
│        └── vpc.yaml
└── templates
    ├── subnets.py
    └── vpc.py
```

A user can delete the Subnets stack using `sceptre delete dev/subnets.yaml`. An important thing to note is that doing so will delete **all** stacks the Subnet is dependent on, this is recursive until there are no more dependent stacks.

Example: The Subnet stack is dependent on the VPC stack, which is dependent on an infrastructure stack, which is finally dependent on an environment stack.

Deleting the Subnet stack will cause the VPC, infrastructure and environment stacks to be deleted.

Reply:
Agreed - I will wait until we have a definitive delete behaviour and then add a note about deletion behaviour.